cmake/modules/FindQatDrv.cmake

```diff
@@ -0,0 +1,122 @@
+# - Find_QatDrv
+# Find the qat driver library and includes
+#
+# QatDrv_INCLUDE_DIRS = where to find cap.h, qae_mem.h, etc
+# QatDrv_LIBRARIES - List of libraries when using qat driver
+# QatDrv_FOUND - True if qat driver found
+
+set(expected_version "v1.7.L.4.14.0-00031")
+set(expected_version_url "https://www.intel.com/content/www/us/en/download/19081/intel-quickassist-technology-intel-qat-driver-for-linux-for-intel-server-boards-and-systems-based-on-intel-62x-chipset.html")
+
+function(find_qatdrv_source_dir)
+  if(NOT DEFINED ENV{ICP_ROOT})
+    set(expected_source_dir "/opt/APP/driver/QAT")
```

Comment:
i'd suggest use `find_path()` and pass `ENV{ICP_ROOT}` and  `/opt/APP/driver/QAT` as `HINTS`